woke.txt

```diff
@@ -21,3 +21,11 @@ pronoun(s)
 meow
 nya+
 nonbinary
+mrrp
+>.<
+uwu
+owo
+:3
+they
```

Comment:
remove they/them because the ordinary 3rd person plural isnt woke...

Reply:
okay but itd be funny imo...